extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1783,7 +1788,18 @@ export class AmpStoryPage extends AMP.BaseElement {
 
       this.mutateElement(() => {
         textEl.textContent = openLabel;
-        this.element.appendChild(this.openAttachmentEl_);
+
+        const container = this.win.document.createElement('div');
+        container.classList.add('i-amphtml-page-attachment-host');
+        container.setAttribute("role", "button");
+        this.element.appendChild(container);
+        createShadowRootWithStyle(container, this.openAttachmentEl_, pageAttachmentCSS);
+
+        const clickTarget = container.shadowRoot.querySelector('.i-amphtml-story-page-open-attachment');
```

Comment:
Isn't this the same as `this.openAttachmentEl_`? You might be able to revert the click listener changes if it works as expected

Reply:
You're totally right. Now that we're setting `role=button` we don't need to do this specific click target in the shadow root.